src/containers/profile/src/caseparticipation/AddPeopleToCaseForm.js

```diff
@@ -0,0 +1,229 @@
+
+  const searchedPeople :List = useSelector((store) => store.getIn([PROFILE, SEARCHED_PEOPLE]));
+  const totalHits :List = useSelector((store) => store.getIn([PROFILE, TOTAL_HITS]));
+  const data = searchedPeople.map((searchedPerson :Map) => {
+    const name = getPersonName(searchedPerson);
+    const personDOB = getPropertyValue(searchedPerson, [DOB, 0]);
+    return Map({ name, person: searchedPerson, personDOB });
+  });
+
+  const searchPeopleRequestState = useSelector((store) => store.getIn([PROFILE, SEARCH_PEOPLE, REQUEST_STATE]));
+  const isSearching :boolean = isPending(searchPeopleRequestState);
+  const hasSearched :boolean = isFailure(searchPeopleRequestState) || isSuccess(searchPeopleRequestState);
+
+  const resetSearchedPeopleList = () => {
+    dispatch(clearSearchedPeople());
+  };
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => resetSearchedPeopleList, []);
+
+  return (
+    <>
+      <CardSegment>
+        <Crumbs>
+          <CrumbItem>
+            <CrumbLink to={`${root}/${personId}`}>
+              <Typography color="inherit" variant="body2">{ personName }</Typography>
+            </CrumbLink>
+          </CrumbItem>
+          <CrumbItem>
+            <Typography color="inherit" variant="body2">Add People to Case</Typography>
+          </CrumbItem>
+        </Crumbs>
+        <Typography variant="h1">Add People to Case</Typography>
+        <Typography gutterBottom>
+          Search for someone to add as a victim or peacemaker to this case.
+        </Typography>
+        <Typography variant="body2" gutterBottom>{ caseIdentifier }</Typography>
+        <CaseRoleTextWrapper>
+          <Typography color="inherit" variant="body2" gutterBottom>{ respondentText }</Typography>
+        </CaseRoleTextWrapper>
+        <CaseRoleTextWrapper>
+          <Typography color="inherit" variant="body2" gutterBottom>{ peacemakerText }</Typography>
+        </CaseRoleTextWrapper>
+        <CaseRoleTextWrapper>
+          <Typography color="inherit" variant="body2" gutterBottom>{ victimText }</Typography>
+        </CaseRoleTextWrapper>
+      </CardSegment>
+      <Card>
+        <CardSegment padding="30px" vertical>
+          <div>Search People</div>
+          <SearchGrid>
+            <div>
+              <Label>First name</Label>
+              <Input
+                  name="firstName"
+                  onChange={onInputChange} />
+            </div>
+            <div>
+              <Label>Last name</Label>
+              <Input
+                  name="lastName"
+                  onChange={onInputChange} />
+            </div>
+            <div>
+              <Label>Date of birth</Label>
+              <DatePicker onChange={(date :string) => setDOB(date)} />
+            </div>
+            <ButtonWrapper>
+              <Button aria-label="Search Button" isLoading={isSearching} onClick={searchPeopleForCase}>Search</Button>
+            </ButtonWrapper>
```

Comment:
Why do you need the button wrapper? I think you can fix the button alignment with CSS in the `SearchGrid` component.